packages/core/build/config.js

```diff
@@ -39,7 +39,7 @@ const copyConfig = base => [
     { from: path.resolve(__dirname, '../node_modules/@deriv/trader/dist/css/**'), to: 'css', flatten: true },
     { from: path.resolve(__dirname, '../node_modules/@deriv/trader/dist/*.*'), to: 'js', flatten: true },
     {
-        from: path.resolve(__dirname, '../node_modules/@deriv/translations/lib/public/i18n/*.*'),
+        from: path.resolve(__dirname, '../../translations/src/translations/*.*'),
```

Comment:
When we refactor these path resolvers to use `require.resolve`, we can fully address this, but for now, can you please use the `node_modules/@deriv/translations` path instead of the path to the package itself? (Like the paths to `@deriv/trader` in the lines above.)

Reply:
@mahdipourziaei-binary `@deriv/translations` will be in the node_modules even though it has no output since the build was removed? (I mean there is no `/dist` or `/lib` anymore)